man/calculate_recommended_interventions.Rd

```diff
@@ -5,121 +5,170 @@
-Each sublist represents a component and contains coefficients for its cost function.
-The position of each coefficient in the sublist corresponds to the power of x in the polynomial cost function.
+\item{cost_list_of_lists}{list: a nested list structure defining cost functions
+for the intervention components. Each sublist represents a component and
+contains coefficients for its cost function. The position of each coefficient
+in the sublist corresponds to the power of x in the polynomial cost function.
 For example:
-list(c(1, 2), c(4), c(5, 4, 3)) represents:
-- First component: cost = 1x + 2x^2
-- Second component: cost = 4x
-- Third component: cost = 5x + 4x^2 + 3x^3
-Empty sublists are not allowed. Each component must have at least one coefficient.}
-
-\item{outcome_goal}{numeric, The outcome goal}
-
-\item{outcome_goal_optimization}{character, Method used behind the scenes for
-calculating the recommended interventions. Either "numerical" or "grid_search"
-For v1, this does not matter since we work with linear cost functions and
-we only have one way of solving for the recommended interventions}
-
-\item{grid_search_step_size}{numeric, step size for the grid search algorithm,
-default value is 0.01 for each intervention component.}
-
-\item{include_confidence_set}{boolean, Whether the confidence set should be
-calculated for the recommended intervention.}
-
-\item{confidence_set_step_size}{numeric vector, Step size for the grid search
-algorithm used in the confidence set calculations.}
-
-\item{confidence_set_alpha}{numeric, Type I error of the confidence set.}
+list(c(1, 2, 3, 4), c(4, 6), c(5, 4, 3)) represents:
+- First component: cost = 1 + 2x_1 + 3x_1^2 + 4x_1^3
+- Second component: cost = 4 + 6x_2
+- Third component: cost = 5 + 4x_3 + 3x_3^2
+list(c(0, 2), c(0, 6), c(0, 4)) represents:
+- First component: cost = 2x_1
+- Second component: cost = 6x_2
+- Third component: cost = 4x_3
+Note: empty sublists are not allowed.}
+
+\item{outcome_goal}{numeric: the outcome goal, a desired probability or mean
+value.}
+
+\item{glm_family}{character: GLM family of the intended outcome model.
+automatically determined if "auto":
+"gaussian" for continuous outcomes, "binomial" for binary outcomes.
+supports "gaussian", "binomial" and "quasibinomial".}
+
+\item{link}{character: the optional link argument for the outcome GLM model.
+automatically determined if "auto":
+"logit" for binary outcomes, and "identity" for continuous outcomes.}
+
+\item{weights}{numeric vector: the optional weights argument when fitting
+GLM of the intended outcome model. Please see help file of the glm function
+from the stats package for details.}
+
+\item{center_characteristic_list}{character vector: a vector of column names
+of the center characteristics. For example: c("characteristic1")}
+
+\item{center_characteristic_list_future}{numeric vector: a vector of values for
```

Comment:
I'm sorry but I'm still confused about why we call it "future" if it's intended for facilities-level. Maybe a different way of explaining it would make it more intuitive? Or if you think the naming is fine, it'd be great if you could explain it to me directly so I can understand it from now on and follow your logic.

Reply:
what about `center_characteristic_list_opt` or `center_characteristic_list_optimization`? these are fixed values that need to provided for LAGO optimization to work. I know you are familiar with Daniel's paper, this would be the `Z=1.75` value that was used during optimization in his paper.